### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'filter')

### DISCUSSION THREAD

**Discussion by oleg-smirnov-qs-pro** [webpack/webpack#16227] [Q&A]
Hi to everyone!

I'm trying to create browser plugin through CRA with TS, so i need to have several folders in `src` and build that folders separately (into several separate `.js` files). 
So i eject CRA, and modify webpack.config to support multiple entry points. 
![image](https://user-images.githubusercontent.com/111351661/188494362-208e7d39-1650-41fb-b606-07addc6edcbb.png)
![image](https://user-images.githubusercontent.com/111351661/188494391-befa2277-79e0-4451-817b-caa299841213.png)

But when i'm trying to build project i've got that error:
```
TypeError: Cannot read properties of undefined (reading 'filter')
    at runMicrotasks (<anonymous>)
    at C:\work\passback-extension\scripts\build.js:176:23
    at finalCallback (C:\work\passback-extension\node_modules\webpack\lib\Compiler.js:441:32)
```
![image](https://user-images.githubusercontent.com/111351661/188494098-fec85c59-3b2a-46c0-98d8-6f308d7a7e3e.png)
![image](https://user-images.githubusercontent.com/111351661/188494171-b1e55ae0-c9d0-43c2-a7cf-bfc4d319f5eb.png)

Can someone suggest, where should i look for reasons of this issue?

Thank you in advance

**Comment by alexander-akait** [comment]
Look at this C:\work\passback-extension\scripts\build.js:176:23

**Reply by oleg-smirnov-qs-pro** [reply]
Thank you for your answer, but `C:\work\passback-extension\scripts\build.js:176:23` just lead to the point where error message concats from `message.errors` list
![image](https://user-images.githubusercontent.com/111351661/188588831-d77314c1-f67f-42ec-a234-032907b6c462.png)

I think the issue happens somewhere here 
`node_modules\webpack\lib\Compiler.js:441:32`
but why `callback !== undefined` (and what is that callback means) - i don't know and i want to find that reason
![image](https://user-images.githubusercontent.com/111351661/188588978-6813c9d6-51d4-49cd-968a-a626a648b375.png)

**Reply by alexander-akait** [reply]
It looks like you are using webpack incorrectly, we don't have bugs here
